util/convert_log.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Comment:
While this (demo) utility have been written before the pipeline implementation, it is very simple and would not benefit much from getting converted to the pipeline interface. It would still be a good first task for someone trying to get acquainted with the pipeline approach.